add settings for user role selection (#76)
darylldoyle/safe-svg: README.md

Safe SVG is the best way to Allow SVG Uploads in WordPress!

> It gives you the ability to allow SVG uploads whilst making sure that they're sanitized to stop SVG/XML vulnerabilities affecting your site.  It also gives you the ability to preview your uploaded SVGs in the media library in all views. It gives you the ability to select which user role can upload an SVG file.

Reviewer: ```suggestion
It gives you the ability to allow SVG uploads (and limit that by user role) whilst making sure that they're sanitized to stop SVG/XML vulnerabilities affecting your site.  It also gives you the ability to preview your uploaded SVGs in the media library in all views.
```
